### PR TITLE
W-15088597-runonce-se

### PR DESCRIPTION
**Pull request by sebermuley**
new run configuration option Execute Manually

### CODE REVIEW

**Review by mseitz97 — DISMISSED** [review]

File: rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc [hunk 1]
```diff
@@ -209,7 +211,13 @@ Bots running an earlier version than 1.2.0 overtake an open console session with
 [[runconfig-execution-enabletaskbar]] If you want to watch the execution of the process in the *Process Streaming* view of the *Process Monitoring* module or if the process uses elements of the Windows operating system like the start menu or the task bar, check *Enable Taskbar (Start menu, taskbar buttons)*.
 * *Process Trigger Execution*:
 +
-The productive run is carried out continuously within a fixed schedule. For each schedule, you can define time intervals within which the process is executed.
+The productive run is carried out continuously within a fixed schedule or just once on the click of a trigger button. 
++
+[[run-once-tag]]If you want to run the process only once on each <<trigger-run-once, click of the trigger button>>, select *Manual Execution*.
```
Comment (Crispy-Salesforce) [inline rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc]:
```suggestion
[[run-once-tag]]If you want to run the process once on each <<trigger-run-once, click of the trigger button>>, select *Manual Execution*.
```

File: rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc [hunk 2]
```diff
@@ -209,7 +211,13 @@ Bots running an earlier version than 1.2.0 overtake an open console session with
...
++
+If you want to schedule the process run, consider and do the following:
++
+For each schedule, you can define time intervals within which the process is executed.
```
Comment (Crispy-Salesforce) [inline rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc]:
```suggestion
For each schedule, you can define time intervals to execute the process.
```

File: rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc [hunk 3]
```diff
@@ -4,7 +4,7 @@ The third phase of process automation is the *Test* phase. In this phase, you ca
 
 When the tests are successful, transition to *Production* phase.
 
-The final phase in the life-cycle of a process is the *Production* phase. This phase is structured similarly to the *Test* phase. The difference is that you do not have to start the individual process runs manually. Instead you schedule a process run configuration which is then executed automatically at defined times. Or you create an invokable run configuration and publish it to Anypoint Exchange.
+The final phase in the life-cycle of a process is the *Production* phase. This phase is structured similarly to the *Test* phase. You create a process run configuration which is then executed at the click of a button or automatically at scheduled times. Or you create an invokable run configuration and publish it to Anypoint Exchange.
```
Comment (Crispy-Salesforce) [inline rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc]:
```suggestion
The final phase in the life cycle of a process is the *Production* phase. This phase is structured similarly to the *Test* phase. You create a process run configuration which is then executed at the click of a button or automatically at scheduled times. Or you create an invokable run configuration and publish it to Anypoint Exchange.
```

File: rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc [hunk 4]
```diff
@@ -209,7 +211,13 @@ Bots running an earlier version than 1.2.0 overtake an open console session with
 [[runconfig-execution-enabletaskbar]] If you want to watch the execution of the process in the *Process Streaming* view of the *Process Monitoring* module or if the process uses elements of the Windows operating system like the start menu or the task bar, check *Enable Taskbar (Start menu, taskbar buttons)*.
 * *Process Trigger Execution*:
 +
-The productive run is carried out continuously within a fixed schedule. For each schedule, you can define time intervals within which the process is executed.
+The productive run is carried out continuously within a fixed schedule or just once on the click of a trigger button.
```
Comment (Crispy-Salesforce) [inline rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc]:
```suggestion
The productive run is carried out continuously within a fixed schedule or just once at the click of a trigger button. 
```

File: rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc [hunk 5]
```diff
@@ -267,6 +275,22 @@ Deploy configuration to run them fully automatically in a productive environment
...
+
+The configuration to run once must either be invocable or marked as <<run-once-tag, *Manual Execution*>>.
+
+To start the single process run:
+
+. Open the *Process Management* view of the *Process Automation* module and click on the name of the category of your project.
+. Click on the name of a project to open it.
+. Click *Select action*.
+. From the dropdown, select *Manual execution*.
+. In the window *Select Configuration to Manual execution*, check all configurations you want to run.
```
Comment (Crispy-Salesforce) [inline rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc]:
```suggestion
. In the window *Select Configuration to Manual execution*, check all configurations to run.
```

File: rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc [hunk 6]
```diff
@@ -267,6 +275,22 @@ Deploy configuration to run them fully automatically in a productive environment
 . In the window *Select Configuration to deploy*, check all configurations you want to run.
 . Click *Execute*.
 
+[[trigger-run-once]]
+=== Run a Configuration Once
+
+The configuration to run once must either be invocable or marked as <<run-once-tag, *Manual Execution*>>.
```
Comment (Crispy-Salesforce) [inline rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc]: "invokable" or "invocable"? Please, use one spelling. Thanks!

**Review by Crispy-Salesforce — DISMISSED** [review]

File: rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc (hunk 2 above)
Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc]: Th process is executed continuously within the interval.

Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc]: -> Added 'continuously' to clarify.

File: rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc (hunk 6 above)
Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc]: At the moment the product still uses invokable. Although it has been recommended to replace this with invocable.

**Review by Crispy-Salesforce — DISMISSED** [review]

**Review by bjoson-mulesoft — APPROVED** [review]

**Comment by sebermuley** [comment]
replaced with New Scheduling PR

**Pull request closed without merging**